ayayaa-server/ayayaa-logging/LoggingServer.cs

```diff
@@ -0,0 +1,177 @@
+﻿using ayayaa.logging.Enums;
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Sockets;
+using System.Text;
+using System.Threading;
+
+namespace ayayaa.logging
+{
+    public class LoggingServer : IDisposable
+    {
+        public LoggingServer(string ip, int port) 
+        {
+            LocalLogger = new Logger();
+
+            this.port = port;
+            IPAddress.TryParse(ip, out this.ip);
+        }
+
+
+        public Logger LocalLogger { get; set; }
```

Comment:
Public access to a variable that is used in a private thread.
Race condition waiting to happen.